dlib/test/image_load.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (C) 2019  Paul Dreik (github@pauldreik.se)
+// License: Boost Software License   See LICENSE.txt for the full license.
+
+#include "tester.h"
+#include <dlib/image_io.h>
+
+
+// Finding files recursively in the filesystem is easy if C++17 or boost
+// can be used. However, none of those can be required, so instead the posix
+// API is used (dirent.h). If that is not available, there is no recursive finding,
+// instead the input argument is assumed to be a file.
+#if defined(unix) || defined(__unix__) || defined(__unix)
```

Comment:
Use `dlib::get_files_in_directory_tree()` instead

Reply:
Thanks, will fix this!

Reply:
Now pushed.